src/test/java/ru/yandex/praktikum/OrderSpecificUser.java

```diff
@@ -0,0 +1,70 @@
+package ru.yandex.praktikum;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Step;
+import io.qameta.allure.junit4.DisplayName;
+import io.restassured.response.Response;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static ru.yandex.praktikum.GenerateUser.*;
+
+public class OrderSpecificUser {
+    String accessToken;
+    int statusCode;
+    boolean isGet;
+    String message;
+    Response response;
+
+    @Before
+    public void setUp() {
+        StepUser.createUser(RANDOM_EMAIL, RANDOM_PASSWORD, RANDOM_NAME);
+        StepUser.loginUser(RANDOM_EMAIL, RANDOM_PASSWORD, " ");
+    }
+
+    @Test
+    @DisplayName("Получение заказа c авторизацией и с ингредиентами")
+    @Description("Проверка - успешное получение заказа с авторизацией и с добавлением ингредиентов")
+    public void getOrderWithAuthorizationWithIngredients() {
+        accessToken = response.then().log().all().extract().path("accessToken");
```

Comment:
Здесь тест падает с ошибкой:
`java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.then()" because "this.response" is null`